docs/examples/users/user.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: rabbitmq.com/v1alpha1
+kind: User
+metadata:
+  name: user-sample
+spec:
+  name: test-user # name of the user
+  tags:
+  - management # available tags are 'management', 'policymaker', 'monitoring' and 'administrator'
```

Comment:
There needs to be a rabbitmqClusterReference in this too.

Reply:
Updated, thanks for catching it!